src/main/javascript/crypto/src/tn-compat/SignedDevconTicket.ts

```diff
@@ -0,0 +1,29 @@
+import { AsnProp, AsnPropTypes } from "@peculiar/asn1-schema";
+import { ValidityValue, AlgorithmIdentifierASN} from "../asn1/shemas/AuthenticationFramework";
+
+export class DevconTicket {
+    @AsnProp({ type: AsnPropTypes.Utf8String }) public devconId: string;
+    @AsnProp({ type: AsnPropTypes.Integer }) public ticketId: number;
+    @AsnProp({ type: AsnPropTypes.Integer }) public ticketClass: number;
+}
```

Comment:
Can you check main branch for code?
https://github.com/TokenScript/attestation/blob/main/src/main/javascript/crypto/src/asn1/shemas/SignedDevconTicket.ts#L10-L17
its single object , compatible with old and new ticket ASN schema
and Ticket.ts updated to be compatable with ID as integer/string and old and new ticket formats

Reply:
we can merge this branch with main and stay Tickets.ts from main branch. it can read both Ticket formats